cmake/FindPostgreSQL.cmake

```diff
@@ -51,7 +51,11 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-include(${CMAKE_SOURCE_DIR}/cmake/OpenSSL.cmake)
+include(${CMAKE_SOURCE_DIR}/cmake/OpenSSL.cmake OPTIONAL RESULT_VARIABLE file_found)
+
+if(file_found==NOTFOUND) 
+include(${CMAKE_SOURCE_DIR}/../../cmake/OpenSSL.cmake RESULT_VARIABLE file_found)
+endif()
```

Comment:
Not supper happy about this, need to find a better way. This was previously handled by having something like this:

```CMake
list(PREPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/../../cmake)
```

in the extensions, as it's basically pointing to the right directory. However, ${CMAKE_SOURCE_DIR} in FindPostgreSQL.cmake is bypassing this. I think we need to figure out how we can make it `include(OpenSSL)` again.

Reply:
Updated using `CMAKE_CURRENT_LIST_FILE` - which provides the current CMAKE file's location. Irrespective of from where `FindPostgreSQL.cmake` is called, it includes the `OpenSSL.cmake` file